pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -333,6 +347,22 @@ func (plugin *cniNetworkPlugin) deleteFromNetwork(network *cniNetwork, podName s
 	return nil
 }
 
+func (plugin *cniNetworkPlugin) SetPodIPs(podSandboxID kubecontainer.ContainerID, ipAddress string) {
```

Comment:
This function only sets one pod IP address, so it should probably be SetPodIP() (eg, not plural)